GEOSldas_App/ldas_setup

```diff
@@ -168,8 +172,16 @@ class LDASsetup:
         self.ladas_coupling = int(self.rqdExeInp.get('LADAS_COUPLING',0))
         self.adas_expdir =''
         if self.ladas_coupling > 0:
-           assert 'ADAS_EXPDIR'  in self.rqdExeInp, " need ADAS_EXPDIR  in the input file %s" %(self.exeinpfile)
-           self.adas_expdir = self.rqdExeInp['ADAS_EXPDIR']
+           self.adas_expdir = os.path.dirname(self.exphome)
+           self.rqdExeInp[ 'ADAS_EXPDIR'] = self.adas_expdir
+           self.adas_expid = os.path.basename(self.adas_expdir)
+           self.rqdExeInp[ 'MET_TAG' ] = self.adas_expid + '__Nx+-'
```

Comment:
Do we really want to edit ldas_setup with '__Nx+-' knowing that this is already being changed?  Can we go straight to the new "bkg" location of the lfo files?

Reply:
I will leave  Nx+-  option in till we get bkg updates in  and tested